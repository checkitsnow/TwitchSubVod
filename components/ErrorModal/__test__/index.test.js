import renderer from 'react-test-renderer';
import ErrorModal from '@/components/ErrorModal';

it('should render Error Modal correctly', () => {
  const tree = renderer
    .create(<ErrorModal message="The user you just searched doesn't exist" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
