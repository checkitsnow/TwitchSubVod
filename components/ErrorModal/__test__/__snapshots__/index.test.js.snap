// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Error Modal correctly 1`] = `
<div
  className="styles__Container-sc-1363ygh-0 gPcMLO"
>
  <svg
    fill="none"
    height={36}
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 24 24"
    width={36}
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="12"
      cy="12"
      r="10"
    />
    <line
      x1="12"
      x2="12"
      y1="8"
      y2="12"
    />
    <line
      x1="12"
      x2="12.01"
      y1="16"
      y2="16"
    />
  </svg>
  <span>
    The user you just searched doesn't exist
  </span>
</div>
`;
